ui/src/components/RolesTable/index.jsx

```diff
@@ -28,10 +53,12 @@ export default class RolesTable extends Component {
     onPageChange: func.isRequired,
     /** A search term to refine the list of roles */
     searchTerm: string,
+    onDialogActionOpen: func,
```

Comment:
This prop is not checked for null-ness before being used, so it should have `.isRequired` like `onPageChange` does.  That also means you can omit it from `defaultProps`, below.